client/state/data-getters/index.js

```diff
@@ -23,6 +23,11 @@ import {
 import { convertToSnakeCase } from 'state/data-layer/utils';
 import { dummyTaxRate } from 'lib/tax'; // #tax-on-checkout-placeholder
 
+export const getAtURL = url =>
```

Comment:
This could use some tests.

Reply:
I'm not sure which tests to add. Do you have recommendations?

Reply:
Not really - thinking about it further, it seems it's pretty straightforward, and any tests would only be duplicating the function body.

Reply:
we might find some valuable ones later but that's exactly how I was thinking about it